requirements.txt

```diff
@@ -13,3 +13,5 @@ azure-mgmt-storage==11.1.0
 azure-storage-blob
 msrest
 msrestazure
+sshpubkeys
+pycryptodome
```

Comment:
This requirement (`pycryptodome`) is used by the CLI tool (see PR #217)

Reply:
By add it here and not in PR #217 then?

Reply:
Because I did not think about doing this :D Removed it, will add it again in the other PR.